python/ray/scripts/scripts.py

```diff
@@ -177,11 +177,22 @@ def cli(logging_level, logging_format):
     is_flag=True,
     default=False,
     help="do not redirect non-worker stdout and stderr to files")
+@click.option(
+    "--plasma-store-socket-name",
+    is_flag=True,
+    default=False,
+    help="manually specify the name of the socket of the plasma store")
+@click.option(
+    "--temp-dir",
+    is_flag=True,
+    default=None,
+    help="manually specify the root temporary dir the Ray process")
```

Comment:
maybe add a test for some of this stuff in https://github.com/ray-project/ray/blob/93ded5a3d533d4e2902ce17fe8547428b0966f6f/test/multi_node_test.py#L307